src/test/java/org/sagebionetworks/bridge/spring/controllers/CRCControllerTest.java

```diff
@@ -637,27 +659,42 @@ public void postAppointmentFailsWhenLocationCallFails() throws Exception {
         when(mockAccountService.authenticate(any(), any())).thenReturn(account);
         when(mockAccountService.getAccount(ACCOUNT_ID)).thenReturn(account);
         
-        HttpResponse mockResponse = mock(HttpResponse.class);
-        doReturn(mockResponse).when(controller).get(any(), any());
+//        mockGetGeocoding();
```

Comment:
Commented out test code?

Reply:
Geocoding is temporarily disabled because it has been reprioritized for launch, but we may restore it so I'm just commenting it out for the moment.